code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -244,6 +266,20 @@
 		if (HH.disease == 1 && HH.disease_type == "typhus")
 			HH.disease_treatment = TRUE
 
+/datum/reagent/saline_glucose
+	name = "Saline Glucose solution"
+	id = "saline_glucose"
+	description = "Saline is a mixture of sodium chloride in water and has a number of uses in medicine."
+	taste_description = "sweetness and salt"
+	reagent_state = LIQUID
+	color = "#dfdfdf8c"
```

Comment:
uh something is wrong here, color string is too long

Reply:
This is because I made color half transparent

Reply:
Did you test it?

Reply:
yes